src/styles/blocks/catalog.scss

```
@@ -1,3 +1,5 @@
+$hover-text: #34568b;
```

```suggestion
$hover-text-color: #34568b;
```
It is better to name it the way it's understandable that it is a color